add-udev-rule.sh

```diff
@@ -10,9 +10,21 @@ source /etc/profile
 # `jq -sR . < rulefilename` and replace the outside the outside double quotes " with single quotes '
 RULE=''
 
+# Finish up the script
+# If message passed as an argument, that means failure.
+finish_up() {
+  local message=$1
+  if [ -n "${message}" ]; then
+    echo "FAIL: ${message}"
+    exit 1
+  else
+    echo "DONE"
+    exit 0
```

Comment:
I don't love forcing an exit 0, as in this case it'll mask the result of the `systemctl restart os-udevrules`. If that command fails, it may not be clear that the change hasn't been applied.

Reply:
@xginn8 this exit 0 is needed by our ssh connection, otherwise it will just keep hanging.

Is it masked, though? I think due to `set -o errexit` it would exit with an error code, it's just not explicitly handled. I'll add one of that as well. See if the newest format is better?